GeneratorInterface/Hydjet2Interface/test/Hydjet2Analyzer.cc

```diff
@@ -570,25 +633,119 @@ void Hydjet2Analyzer::analyze(const edm::Event &iEvent, const edm::EventSetup &i
 
           if (doHistos_) {
             //common histos
-            if ((*it)->status() < 10) {  //status 1
+            if ((*it)->status() == 1) {
+              dhpdg->Fill(pdg_id);
               dheta->Fill(eta);
 
+              e_sum += e;
+              et_sum += et;
+              if (std::abs(eta) < 1.5) {
```

Comment:
if you define (e.g.) `etaMaxBarrel` , `etaMaxAlice` , `etaMaxCms` it is easier to code, and modify afterwards would you ever need it

Reply:
Well, firstly we have to change detector then)) I thing it is useless to introduce variables like etaMaxBarrel. I prefer to leave it hardcoded in default section, to vary cuts from the configs, it is found userHistos_ section, see [this line](https://github.com/cms-sw/cmssw/blob/a3b55905708b74b0fdc0b65f6484d58d51310e99/GeneratorInterface/Hydjet2Interface/test/Hydjet2Analyzer.cc#L762)